frontend/src/components/AppDateInput.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class="text_label">
+    <label>{{label}}</label>
+  </div>
+  <div v-bind="$attrs">
+    <Datepicker
+      :autoApply="true"
+      :class="inputClass"
+      :enableTimePicker="false"
+      :format="inputDateFormat"
+      :text-input="true"
+      v-model="value" 
+    />
+  </div>
+  <AppInputError :e-model="eModel" :label="label"/>
+
+</template>
+
+<script>
+import Datepicker from 'vue3-date-time-picker';
+import AppInputError from './AppInputError.vue'
+import 'vue3-date-time-picker/dist/main.css'
+import { INPUT_DATE_FORMAT } from '../util/constants.js'
+
+  export default {
+    name: 'AppDateInput',
+    components: {AppInputError, Datepicker},
+    created() {
+      this.inputDateFormat = INPUT_DATE_FORMAT
+    },
```

Comment:
I think this might be a more _standard-ish_ way to just assign a value to a variable?
```
    data() {
      return {
        inputDateFormat: INPUT_DATE_FORMAT
      }
    },
```